src/PathMappingRepository.php

```diff
@@ -260,6 +264,39 @@ private function resolveFilesystemPaths($path, $onlyFirst = true)
     }
 
     /**
+     * If the resource has a parent that's a symbolic link to a directory, then
+     * I'll resolve it and append the filesystem part.
+     *
+     * @param string $path      The repository path.
+     * @param bool   $onlyFirst Should the method stop on the first path found?
+     *
+     * @return string[]|null[]
+     */
+    private function mergeMixedParent($path, $onlyFirst = true)
+    {
+        $resolvedFilesystemPaths = array(null);
+        $parentPath = $path;
+
+        for ($i = 0; $i < substr_count($path, '/') - 1; ++$i) {
+            $parentPath = dirname($parentPath);
+
+            if ($this->store->exists($parentPath)) {
+                $resolvedFilesystemPaths = $this->resolveFilesystemPaths($parentPath, $onlyFirst);
+
+                break;
```

Comment:
I don't think you should beark here: you should break when you resolved a valid resource, ie when `resolvedFilesystemPaths` returned something valid :) . Technically, you can have multiple parents.